tests/test_enki.py

```diff
@@ -297,18 +314,19 @@ def test_get_all_titles(self):
         [method, url, headers, data, params, kwargs] = self.api.requests.pop()
         eq_("get", method)
         eq_("https://enkilibrary.org/API/ListAPI", url)
-        eq_("c", params['lib'])
         eq_("getAllTitles", params['method'])
         eq_("secontent", params['id'])
 
+        # Unlike some API calls, it's not necessary to pass 'lib' in here.
+        assert 'lib' not in params
+
     def test__epoch_to_struct(self):
         """Test the _epoch_to_struct helper method."""
         eq_(datetime.datetime(1970, 1, 1), EnkiAPI._epoch_to_struct("0"))
 
-    def test_checkout_open_access(self):
+    def test_checkout_open_access_parser(self):
```

Comment:
These tests tested the parser for the result of the API call, but not what came in and went out of the API calls themselves, so I had to add some more test coverage (e.g. test_checkout_success below).